custom/icds_reports/migrations/sql_templates/database_views/disha_indicators_v2.sql

```diff
@@ -0,0 +1,35 @@
+DROP VIEW IF EXISTS icds_disha_indicators_v2 CASCADE;
+CREATE VIEW icds_disha_indicators_v2 AS
+SELECT
+    "awc_monthly"."state_id" AS "state_id",
+    "awc_monthly"."state_name" AS "state_name",
+    "awc_monthly"."aggregation_level" AS "aggregation_level",
```

Comment:
If this will always be `1` im not sure it is needed

Reply:
yeah we dont need it. Will remvoe this